django_scrubber/management/commands/scrub_data.py

```diff
@@ -75,6 +83,14 @@ def handle(self, *args, **kwargs):
             except DataError as e:
                 raise CommandError('DataError while scrubbing %s (%s)' % (model, e))
 
+        # Truncate session data
+        if not kwargs.get('keep_sessions', True):
+            Session.objects.all().delete()
+
+        # Truncate Faker data
+        if not kwargs.get('remove_fake_data', False):
```

Comment:
I think the tests are breaking because the logic here is inverted:
```suggestion
        if kwargs.get('remove_fake_data', False):
```

We're deleting the faker data after a run, so the "two runs" test doesn't have the data anymore.

Reply:
I fixed it. Actually, I want sessions to be removed by default - so I fixed both statements.